src/Util/Common.php

```diff
@@ -83,7 +83,7 @@ public static function isReadable($path)
      *
      * @param string $path The path to use.
      *
-     * @return mixed
+     * @return array|false|string|string[]
```

Comment:
```suggestion
     * @return string|false
```

What are the other types based on ? I can't find a code path which would lead to those....